falcon/util/misc.py

```diff
@@ -210,3 +212,25 @@ def get_bound_method(obj, method_name):
             raise AttributeError(msg)
 
     return method
+
+
+def get_http_status(status_code):
+    """Gets both the http status code and description from just a code
+
+    Args:
+        status_code: integer or string that can be converted to an integer
+
+    Returns:
+        str: status code e.g. "404 Not Found"
+
+    Raises:
+        ValueError: the value entered could not be converted to an integer
+
+    """
+    code = int(status_code)  # converts to integer or raises ValueError
+
+    try:
+        return getattr(falcon.status_codes, 'HTTP_' + str(code))
+    except AttributeError:
+        # not found
+        return str(code) + ' Unknown Status Code'
```

Comment:
1) why use str(code) when you can use status_code, which is already a string
2) why do you want to raise ValueError in this function?